Tasks/distinct.js

```diff
@@ -1,19 +1,16 @@
 // Return an array without duplicates
 
-DISTINCT = (data) => {
-  console.log({ data });
-  A = new Set();
-  w = 0;
-  data.forEach((a) => {
-    if (A.has(a)) {
-      delete data[w];
-    } else {
-      A.add(a);
-    }
-    w++;
-  });
-  return data.filter
-  (x => typeof x === 'number');
+"use strict";
+
+const distinct = (array) => {
```

Comment:
It's a matter of style, so it looks better without the extra brackets
```suggestion
const distinct = array => {
```

Reply:
Totally agree with you, but in this case it is better to use brackets, as stated in HINTS.md - "Use round brackets even for single argument lambda functions"

https://github.com/HowProgrammingWorks/Exams/blob/master/HINTS.md